Cubical/Algebra/Module/Instances/FinMatrix.agda

```diff
@@ -0,0 +1,39 @@
+{-# OPTIONS --safe #-}
+module Cubical.Algebra.Module.Instances.FinMatrix where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.SIP
+
+open import Cubical.Functions.FunExtEquiv using (funExt₂)
+
+open import Cubical.Data.Nat using (ℕ)
+
+open import Cubical.Algebra.Matrix
+open import Cubical.Algebra.Ring
+open import Cubical.Algebra.Module
+
+module _ {ℓ} (R : Ring ℓ) {m n : ℕ} where
+
+  open module R' = RingStr (str R) renaming (_+_ to _+r_; -_ to -r_)
+  open LeftModuleStr
+
+  FinVecLeftModule : LeftModule R ℓ
+  fst FinVecLeftModule = FinMatrix ⟨ R ⟩ m n
```

Comment:
Since 'FinMatrix' expands to a 'FinVec' of 'FinVec's, you could probably just generalize the work from your last PR, to 'FinVec's of elements of a 'LeftModule' and apply it here. Have you tried that?

Reply:
I made some generalizations now, but I don't know if it is possible to do more.